src/components/Footer/index.jsx

```diff
@@ -1,7 +1,41 @@
 import "./style.css";
+import discord from "../../assets/discord.svg";
+import facebook from "../../assets/fb_icon.svg";
+import instagram from "../../assets/insta_icon.svg";
+import mail from "../../assets/mail_icon.svg";
 
 const Footer = () => {
-  return <div></div>;
+  return (
+    <div id="footer">
+      <div>
+        <h1 id="h1">
```

Comment:
use more descriptive css selectors please (change to #footer-header or something)